lib/live_activities.dart

```diff
@@ -41,18 +41,22 @@ class LiveActivities {
     );
   }
 
-  /// Update an iOS 16.1+ live activity.
+  /// Update an iOS 16.2+ live activity.
   /// You can get an activity id by calling [createActivity].
   /// Data is a map of key/value pairs that will be transmitted to your iOS extension widget.
   /// Map is limited to String keys and values for now.
+  ///
+  /// [StaleIn] indicates if a StaleDate should be added to the activity. If the value is null or the Duration
+  /// is less than 1 minute then no staleDate will be used. The parameter only affects the live activity on
+  /// iOS 16.2+ and does nothing on on iOS 16.2
   Future updateActivity(String activityId, Map<String, dynamic> data,
-      [AlertConfig? alertConfig]) async {
+      {Duration? staleIn, AlertConfig? alertConfig}) async {
     await _appGroupsImageService.sendImageToAppGroups(data);
-    return LiveActivitiesPlatform.instance
-        .updateActivity(activityId, data, alertConfig);
+    return LiveActivitiesPlatform.instance.updateActivity(activityId, data,
+        alertConfig: alertConfig, staleIn: staleIn);
   }
 
-  /// End an iOS 16.1+ live activity.
+  /// End an iOS 16.2+ live activity.
```

Comment:
Maybe we can remove all "iOS X" version label from comments on this file?

Reply:
Done! Should I "resolve" these?